File: k-kbk/1주차/2480.py
from collections import Counter

numbers = list(map(int, input().split(" ")))

# most_common(): 많이 나온 순으로 정렬
counter = Counter(numbers).most_common()

size = len(counter)

if size == 1:
    print(10_000 + numbers[0] * 1_000)
elif size == 2:
    number = counter[0][0]
    print(1_000 + number * 100)
else:
    print(max(numbers) * 100)

File: keemsebin/1주차/2480.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let input = [];

rl.on("line", function (line) {
  input.push(line);
}).on("close", function () {
  let [first, second, third] = input[0].split(" ").map(Number);

  // 1. 세가지 수가 모두 같은 경우
  if (first === second && second === third) {
    console.log(10000 + first * 1000);
  }
  // 2. 두 수만 같은 경우
  else if (first === second || first === third) {
    console.log(1000 + first * 100);
  } else if (second === third) {
    console.log(1000 + second * 100);
  }
  // 3. 모두 다른 경우
  else {
    console.log(Math.max(first, second, third) * 100);
  }
});
